docs/user/manual.adoc

```diff
@@ -232,6 +232,26 @@ The `rust-analyzer` binary can be installed via https://brew.sh/[Homebrew].
 $ brew install rust-analyzer
 ----
 
+=== VS Code or VSCodium in Flatpak
+
+We do not recommend using `rust-analyzer` with a Flatpak version of Code.
+Flatpak applications are sandboxed, which has some unfortunate downsides.
+While the sandbox can be disabled for some directories, `/usr/bin` will always be mounted under `/run/host/usr/bin`.
+This prevents applications from running the system's C compiler, or a system-wide installation of Rust.
```

Comment:
The `org.freedesktop.Sdk` runtime (which both vscode and vscodium has a dependency on) contains a C compiler. There is also the `org.freedesktop.Sdk.Extension.rust-stable` sdk extension containing rustc, but that one you need to install manually. I personally use vscode in flatpak with a rustup installed rustc without issues every day. The only time I had trouble was when I used rust-bindgen as libclang is missing, but I just found that `org.freedesktop.Sdk.Extension.llvm15` exists which I think has libclang.

Reply:
If everything is great and clear, then what is the issue behind #14042 and #13926?

Reply:
I'm using a rustup installed rustc and I've configured the `PATH` env var to add `/home/bjorn/.cargo/bin:` before the default value. Other than that I don't have any flatpak overrides. I also couldn't find any relevant differences between the vscode and vscodium flatpak.

Reply:
@DanielProg39 in #14042, @ionutrogojan was using Rust installed with `rustup`, but without adding it to Codium' `PATH`.

I assume you were in the same situation in #13926.

The "Flatpak-native" solution would be to uninstall `rustup` and Rust, and install `org.freedesktop.Sdk.Extension.rust-stable` from Flatpak.